src/xpk/parser/batch.py

```diff
@@ -39,3 +41,157 @@ def set_batch_parser(batch_parser):
   add_shared_arguments(batch_optional_arguments)
 
   batch_parser.set_defaults(func=batch)
+
+  batch_optional_arguments.add_argument(
+      '--ignore-unknown-flags',
+      type=bool,
+      action=argparse.BooleanOptionalAction,
+      default=False,
+      help='Ignore all the unsupported flags in the bash script.',
+  )
+  batch_optional_arguments.add_argument(
+      '--first-node-ip',
+      type=bool,
+      action=argparse.BooleanOptionalAction,
+      default=False,
+      help="Enable the retrieval of the first node's IP address.",
+  )
+  # TODO: Find solution for time.Duration()
+  # batch_optional_arguments.add_argument(
+  #   '--first-node-ip-timeout',
+  #   type=int,
+  #   default=False,
+  #   help='The timeout for the retrieval of the first node\'s IP address.'
```

Comment:
I also can't find Duration type for this flag. Maybe we should set as string and add custom validation. 

@IrvingMg @pawloch00 WDYT?

Reply:
Removed